cmc/values.yaml

```diff
@@ -0,0 +1,239 @@
+tags:
+  cmc-claim-store-postgresql-pod: false # claim-store chart
+  cmc-citizen-frontend-pod: false       # claim-store chart
+  cmc-ccd-stack: true                   # claim-store chart
+  idam-pr: true                         # citizen-frontend chart - will add IDAM redirect urls
+  ccd-idam-pr: true                     # claim-store chart - will add IDAM redirect urls
+  rpe-feature-toggle-api-pod: true      # citizen-frontend dep
+  postgresql-pod: false                 # any chart specific postgres
+  blobstorage: false                    # blob storage for dm-store
+
+### CMC SERVICES ###
+cmc-claim-store:
+  java:
+    image: hmcts.azurecr.io/hmcts/cmc-claim-store:latest
+    readinessDelay: 180
+    livenessDelay: 180
+    readinessPeriod: 60
+    livenessPeriod: 60
+    keyVaults:
+    environment:
+      CLAIM_STORE_DB_USERNAME: hmcts
+      CLAIM_STORE_DB_PASSWORD: hmcts
+      CMC_DB_USERNAME: hmcts
+      CMC_DB_PASSWORD: hmcts
+  ccd:
+    readinessDelay: 180
+    livenessDelay: 180
+    readinessPeriod: 60
+    livenessPeriod: 60
+    adminWeb:
+      enabled: true
+      environment:
+        NODE_TLS_REJECT_UNAUTHORIZED: 0
+    caseManagementWeb:
+      enabled: true
+      environment:
+        NODE_TLS_REJECT_UNAUTHORIZED: 0
+    importer:
+      userprofile:
+        enabled: true
+        jurisdictions:
+          - CMC
+        users:
+          - civilmoneyclaims+ccd@gmail.com|CMC|MoneyClaimCase|open
+        userProfileDatabasePort: 5432
+        userProfileDatabaseUser: hmcts
+        userProfileDatabasePassword: hmcts
+        userProfileDatabaseName: user-profile
+      definition:
+        enabled: true
+        image: hmcts.azurecr.io/hmcts/cmc-ccd-definition-importer:1.2.6.AKS
```

Comment:
is this not in hmctspublic?

Reply:
nope - im adding a ticket to check - i think there is a handful of repos that can be moved tbh